src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -1910,5 +1917,170 @@ public void testCanHandleSymbollicAlleleFuncotations() {
             Assert.assertEquals(funcotations.get(0).getField("Gencode_19_variantClassification"), expected[2]);
         }
     }
-}
+
+    @DataProvider
+    public Object[][] provideForCheckIfAlreadyAnnotatedTest() {
+        return new Object[][] {
+                {
+                    FuncotatorTestConstants.NON_TRIVIAL_DATA_VALIDATION_TEST_HG19_DATA_SET_1,
+                        b37Reference,
+                        FuncotatorTestConstants.REFERENCE_VERSION_HG19,
+                        FuncotatorTestConstants.FUNCOTATOR_DATA_SOURCES_MAIN_FOLDER,
+                        FuncotatorTestConstants.NON_TRIVIAL_DATA_VALIDATION_TEST_HG19_DATA_SET_1_EXPECTED_OUTPUT
+                }
+        };
+    }
+    /**
+     * An integration test for the Funcotator tool, and specifically,
+     * for the function checkIfAlreadyAnnotated which is included in this
+     * tool. In this test, we are giving as input a vcf which has not already been
+     * annotated, so there should be no problems.
+     * Created by Hailey on 7/9/21.
+     */
+    @Test(dataProvider = "provideForCheckIfAlreadyAnnotatedTest")
+    public void testCheckIfAlreadyAnnotatedIntegrated(final String inputVcfName,
+                                                      final String referencePath,
+                                                      final String referenceVersion,
+                                                      final String dataSourcesPath,
+                                                      final String expectedOutputPath ) {
+
+        for ( final FuncotatorArgumentDefinitions.OutputFormatType outputFormatType :
+        FuncotatorArgumentDefinitions.OutputFormatType.values()) {
+            if (outputFormatType.equals(FuncotatorArgumentDefinitions.OutputFormatType.SEG)) {
+                continue;
+            }
+            final String typeCorrectedExpectedOutPath = FilenameUtils.removeExtension(expectedOutputPath) +
+                    "." + outputFormatType.toString().toLowerCase();
+            final File outputFile = createTempFile(tmpOutDir + File.separator + inputVcfName + ".funcotator", "." + outputFormatType.toString().toLowerCase());
+
+            final ArgumentsBuilder arguments = createBaselineArgumentsForFuncotator(
+                    inputVcfName,
+                    outputFile,
+                    referencePath,
+                    dataSourcesPath,
+                    referenceVersion,
+                    outputFormatType,
+                    true);
```

Comment:
This "positive" test case on an unannotated file is only worth having if you turn on the `--reannotate-vcf` argument, to make sure that that argument doesn't cause a failure with an unannotated file. But as far as I can tell, it's off here.